tests/user/add_remove_credentials.robot

```diff
@@ -15,6 +15,7 @@ ${TEST_OIDC_SUBJECT}  Test OIDC subject
 
```

Comment:
This is another fix that could be reported within the final merge/rebase commit description